tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -5,70 +5,96 @@
+    ['button-press-topdown-v1', SawyerButtonPressTopdownV1Policy(), .0, 1.],
+    ['button-press-v1', SawyerButtonPressV1Policy(), 0., 0.94],
+    ['door-open-v1', SawyerDoorOpenV1Policy(), .0, 0.99],
+    ['door-close-v1', SawyerDoorCloseV1Policy(), .0, 0.99],
+    ['drawer-open-v1', SawyerDrawerOpenV1Policy(), .0, 0.99],
+    ['drawer-close-v1', SawyerDrawerCloseV1Policy(), .0, 0.99],
+    ['lever-pull-v2', SawyerLeverPullV2Policy(), .0, 1.],
+    ['plate-slide-back-side-v2', SawyerPlateSlideBackSideV2Policy(), .0, 1.],
+    ['plate-slide-back-v1', SawyerPlateSlideBackV1Policy(), .0, 1.],
+    ['plate-slide-side-v1', SawyerPlateSlideSideV1Policy(), .0, 1.],
+    ['plate-slide-v2', SawyerPlateSlideV2Policy(), .0, 1.],
+    ['reach-v2', SawyerReachV2Policy(), .0, .99],
+    ['push-v2', SawyerPushV2Policy(), .0, .99],
+    ['pick-place-v2', SawyerPickPlaceV2Policy(), .0, .96],
+    ['basketball-v2', SawyerBasketballV2Policy(), .0, .99],
+    ['peg-insert-side-v2', SawyerPegInsertionSideV2Policy(), .0, .94],
+    ['peg-unplug-side-v1', SawyerPegUnplugSideV1Policy(), .0, .99],
+    ['sweep-into-v1', SawyerSweepIntoV1Policy(), .0, 1.],
+    ['sweep-v1', SawyerSweepV1Policy(), .0, 1.],
+    ['window-open-v2', SawyerWindowOpenV2Policy(), 0., 0.96],
+    ['window-close-v2', SawyerWindowCloseV2Policy(), 0., 0.98],
+]
+
+test_cases_latest_noisy = [
+    # name, policy, action noise pct, success rate
+    ['button-press-topdown-v1', SawyerButtonPressTopdownV1Policy(), .1, .99],
+    ['button-press-v1', SawyerButtonPressV1Policy(), 0., 0.94],
+    ['door-open-v1', SawyerDoorOpenV1Policy(), .1, 0.96],
+    ['door-close-v1', SawyerDoorCloseV1Policy(), .1, 0.99],
+    ['drawer-open-v1', SawyerDrawerOpenV1Policy(), .1, 0.97],
+    ['drawer-close-v1', SawyerDrawerCloseV1Policy(), .1, 0.75],
+    ['lever-pull-v2', SawyerLeverPullV2Policy(), .1, 1.],
+    ['plate-slide-back-side-v2', SawyerPlateSlideBackSideV2Policy(), .1, 0.97],
+    ['plate-slide-back-v1', SawyerPlateSlideBackV1Policy(), .1, .96],
+    ['plate-slide-side-v1', SawyerPlateSlideSideV1Policy(), .1, .82],
+    ['plate-slide-v2', SawyerPlateSlideV2Policy(), .1, .99],
+    ['reach-v2', SawyerReachV2Policy(), .1, .99],
+    ['push-v2', SawyerPushV2Policy(), .1, .97],
+    ['pick-place-v2', SawyerPickPlaceV2Policy(), .1, .92],
+    ['basketball-v2', SawyerBasketballV2Policy(), .1, .99],
+    ['peg-insert-side-v2', SawyerPegInsertionSideV2Policy(), .1, .93],
+    ['peg-unplug-side-v1', SawyerPegUnplugSideV1Policy(), .1, .98],
+    ['sweep-into-v1', SawyerSweepIntoV1Policy(), .1, 1.],
+    ['sweep-v1', SawyerSweepV1Policy(), .1, 1.],
+    ['window-open-v2', SawyerWindowOpenV2Policy(), .1, 0.96],
+    ['window-close-v2', SawyerWindowCloseV2Policy(), .1, 0.96],
+]
+
+
+# Combine test cases into a single array to pass to parameterized test function
+test_cases = []
+for row in test_cases_old_nonoise:
+    test_cases.append(pytest.param(*row, marks=pytest.mark.deprecated))
+for row in test_cases_old_noisy:
```

Comment:
why not just have 4 different parameterized test cases? it's easier to read and maintain.

Reply:
As far as I know, when the tests are parameterized, each test case has to have its own mark (not just the test as a whole). So that means we’d have to do these loops anyway. But maybe there’s a way of stacking the `@parameterized` and `@mark` decorators? I can look into it.

Reply:
I searched for a better way of doing this but didn't find one. Stacking the parameterized decorators just makes pytest complain about the fact that parameter names are listed twice.

Reply:
you can definitely just mark an entire parameterized test. see for instance https://github.com/rlworkgroup/garage/blob/179d49ddccd1316698a4fb3d69cbaf5622cf7564/tests/garage/tf/envs/test_base.py#L26

Reply:
Yes, but I don't think I can stack the parameterized decorator. So the only thing we could do is to write 4 functions (e.g. `test_old_nonoise(...)`, `test_old_noise(...)`, etc.) and then parameterize and mark each one. Is that preferable to combining the test cases with individual marks?

Reply:
doesn't the example i sent involve a stacked decorator?

Reply:
Yes. When I tested it, stacking `@mark` with `@parameterized` works, but stacking two `@parameterized` doesn't work. It complains that the function arguments are specified more than once (once in the first decorator, and again in the second). This is why we would have to create 4 separate functions--that way each one could just have a single `@mark` and a single `@parameterized`